examples/tv-casting-app/tv-casting-common/core/CastingPlayer.cpp

```diff
@@ -272,6 +258,8 @@ CHIP_ERROR CastingPlayer::SendUserDirectedCommissioningRequest()
 
     chip::Protocols::UserDirectedCommissioning::IdentificationDeclaration id = mIdOptions.buildIdentificationDeclarationMessage();
 
+    ReturnErrorOnFailure(support::ChipDeviceEventHandler::SetUdcStatus(true));
```

Comment:
Why is this needed?

Reply:
In my previous PR I moved this from here to VerifyOrEstablishConnection and ContinueConnecting but it's more efficient to have it here once. It previously wasn't called when VerifyOrEstablishConnection was called with CommissionerPasscode set to true for the Commissioner-Generated passcode commissioning flow.
https://github.com/project-chip/connectedhomeip/pull/33479/files#diff-a4c1c744f46eb83abef9f14adb8b60b2e53d0dc8e0d4b846fab4860fbe36b4dfL174